src/redismodule.h

```diff
@@ -1,5 +1,5 @@
-#ifndef REDISMODULE_H
-#define REDISMODULE_H
+#ifndef VALKEYMODULE_H
+#define VALKEYMODULE_H
```

Comment:
Maybe this file is an exception. It has all the function and struct names from Redis to be compatible with Redis module API. It can also keep this guard IMO.

```suggestion
#ifndef REDISMODULE_H
#define REDISMODULE_H
```